code-challenges.js

```diff
@@ -107,34 +110,86 @@
-
-//The .split("") method will do. This will split all the characters in the string into elements of an array.
-
-//The value of our new variable bestCohortof2022ever is now an array.
-
-//Now that we have an array to work with, we can use .reverse() method to reverse the elements in our array.
-
-//The elements in the array are now reversed, however they are all seperated by single quotes since they are elements of an array.
-
-
-
+// Next we need to ffind an array method that will Reverse the values in an array. we can do that by using, .reverse() method. After that I need to find Find an method that will convert thee string "Golf 2022" into an array.
+// In this use case I will use the The .split(""). This will split all the characters in the string into elements of an array.
+// I will Start by creating a new a array. I will name it bestCohortof2022ever.
+// Now that we have an array to work with, we can use .reverse() method to reverse the elements in our array.
+// The elements in the array are now reversed, however they are all seperated by single quotes since they are elements of an array.
+// Lastly, we need to Console log the variable to check the result.
 
 //Find a method that will convert the array back into a string
 
-    // The .join("") method will do just that.
+// const bestCohortof2022ever = "Golf 2022"
+// console.log(bestCohortof2022ever.split(""))
 
 // Console log the variable to check the result.
 
 // const currentCohort = "Golf 2022"
 // // Expected output: "2202 floG"
 
 
-
 // const bestCohortof2022ever = currentCohort.split('').reverse().join("")
 // const currentCohort = "Golf 2022"
 // Expected output: "2202 floG"
 
 
+// --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
+// Describe your process:
+    
+    // 1. Find a method that will go through the array and grab the last of that element we pass in. 
+    
+    // 2. We will Use the .lastIndexOf() method on the array and pass the arguments to Return the last indexs value. This is useful 
+    // when there are repeated values in an array.
+    
+    // 3. We will do all this witin the console.log when calling on out array to pull the desired index.
+    
+        // The ouput should show us the last index or location of the number we pass in the array.
+    
+    
+        // const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
+    
+        // console.log(myNumbers.lastIndexOf(10))  
+        // output: 8
+        
+        // console.log(myNumbers.lastIndexOf(42))
+        // //output: 7
+//Rant: 
+// Aftergoing through this assesment I've realized I keep trying to remember everything and thats not going to happen. I'm familiar with the process and a good portion of the vocabulary. However, I still cannot recall the proper syntax to ultalize I have to look it up.
```

Comment:
For sure, we don't expect you to memorize all of the vocab plus the syntax right away.  The syntax will become part of your memory as you practice more, but the vocab is something to really spend time and get familiar with because this will help you form questions and research the right things.